src/coffee/client.coffee

```diff
@@ -220,9 +220,8 @@ class SphereClient
   #
   # options - An {Object} to configure the client
   constructor: (options = {}) ->
-
     # Private: instance of a {TaskQueue}
-    @_task = options.task or new TaskQueue
+    @_task = options.task or new RepeaterTaskQueue {}, { attempts: 50, timeout: 200, timeoutType: 'v' }
```

Comment:
@hisabimbola I think that there should be some default values because a lot of projects already uses this SDK and we cannot force them to change Client creation code. 
But as I said you can pass there your own repeater.